model/btp_application_certificate.py

```diff
@@ -0,0 +1,90 @@
+from model.base import (
+    NamedModelElement,
+)
+
+
+class BtpApplicationCertificate(NamedModelElement):
+    def subject_template(self) -> str:
+        '''
+        certificate subject template to inject CN
+        '''
+        return self.raw['subject_template']
+
+    def common_name(self) -> str:
+        '''
+        subject common name
+        '''
+        return self.raw['common_name']
+
+    def scopes(self) -> list[str]:
+        '''
+        scopes for certificate
+        '''
+        return self.raw['scopes']
+
+    def subject(self, cn: str) -> str:
+        return self.subject_template().format(cn=cn)
+
+    @staticmethod
+    def get_serial_no_from_common_name(cn: str) -> tuple[int, str]:
```

Comment:
naming-nit: since this method is not a simple getter, but a simple parser, I would rather choose `parse_..` as a verb

it would be nice to briefly explain the parsed/expected format in method's docstr